index.html

```diff
@@ -180,11 +225,13 @@ <h2>Skills</h2>
         </div>
       </div>
     </section>
-    <section class="contact">
-      <p>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </p>
+    <section class="contact" id="contact-link">
+      <div class="last-p">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
```

Comment:
On the desktop version, your font properties are making your project look a little bit different from the design, kindly make adjust them, so your texts look like the given design🤗

your app
<img width="866" alt="your_font" src="https://user-images.githubusercontent.com/64643995/213414320-9afca229-89c1-40be-bc4e-371047316922.png">

design
<img width="464" alt="desin_font" src="https://user-images.githubusercontent.com/64643995/213414873-1e50949f-faec-45ab-8ae9-35a90555e3cc.png">